torchaudio/_extension.py

```diff
@@ -45,11 +46,25 @@ def _load_lib(lib: str) -> bool:
             This behavior was chosen because the expected failure case is not recoverable.
             If a dependency is missing, then users have to install it.
     """
-    path = _get_lib_path(lib)
-    if not path.exists():
+    lib_path = _get_extension_path(lib_name)
+    # On Windows Python-3.8+ has `os.add_dll_directory` call,
+    # which is called from _get_extension_path to configure dll search path
+    # Condition below adds a workaround for older versions by
+    # explicitly calling `LoadLibraryExW` with the following flags:
+    #  - LOAD_LIBRARY_SEARCH_DEFAULT_DIRS (0x1000)
+    #  - LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR (0x100)
+    if os.name == "nt" and sys.version_info < (3, 8):
+        _kernel32 = ctypes.WinDLL("kernel32.dll", use_last_error=True)
+        if hasattr(_kernel32, "LoadLibraryExW"):
+            _kernel32.LoadLibraryExW(lib_path, None, 0x00001100)
+        else:
+            warn("LoadLibraryExW is missing in kernel32.dll")
```

Comment:
Can we mention that this is a copy-n-paste from respective TorchVision file? And perhaps just move it to core PyTorch, as torchtext can run into this one as well at some point